vpc/tool/container2/setup_container_linux.go

```diff
@@ -694,17 +671,22 @@ func deleteFromIPv4BPFMap(ctx context.Context, assignment *vpcapi.AssignIPRespon
 	defer unix.Close(int(fd))
 	runtime.KeepAlive(path)
```

Comment:
On a second pass, if you wanted to, I think you could DRY out the raw bpf map stuff with a function like:

    deleteKeyFromBPFMap(mapPath string, key []byte)

that would do the fd BPF_OBJ_GET, uba, BPF_MAP_DELETE_ELEM, and return just the error.

and make `"/sys/fs/bpf//tc/globals/ipv4_map\000"` consts 

Maybe. We are talking 2 copies of the code here, and my goto for refactoring is 3 copies. Up to you.